### PR TITLE
Repo configurations branch

### PR DESCRIPTION
**Pull request by Mjamal-ubuntu**
Adding README file with description & modifiying .gitignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+This Repo is made for learning C++ programming language, and practicing on all the differencies between C & C++ 
+Level 2 folder is for the playlist Level2 on youtube for Moatassem. 
+hacckerrank folder will be made to solve all the problems in hacckerrank site.
```
Comment (gemy-97) [inline README.md]: I see another PR creating the hacker rank folder so, I think this file will need to be updated within the second PR.

**Review by gemy-97 — CHANGES_REQUESTED** [review]
I think we need to align with each other, how the README file will be updated in organized way.

File: README.md (hunk 1 above)
Comment (Mjamal-ubuntu) [reply README.md]: done , the README file updated

**Review by gemy-97 — APPROVED** [review]
looks fine to me , approved

**Comment by gemy-97** [comment]
PR done

**Pull request merged**